Seohee/Lv2/위장.js

```diff
@@ -0,0 +1,19 @@
+function solution(clothes) {
+    let result = 1;
+    const clothesType = {};
```

Comment:
```suggestion
    const clothesType = {};
    let result = 1;
```
일반적으로 변수 선언할 때 현업에서 const를 먼저 선언한다고 하더라고요